src/routes/consent.ts

```diff
@@ -247,6 +247,7 @@ export const registerConsentRoute: RouteRegistrator = function (
     console.log("found HYDRA_ADMIN_URL")
     return app.get(
       "/consent",
+      // @ts-ignore
```

Comment:
why do we need ts-ignore? can we maybe remove it instead and either use the correct type or cast it?